vue3-apollo-quasar/src/components/PaginationButtons/data.ts

```diff
@@ -0,0 +1,5 @@
+export const PAGINATION_EVENT = 'paginate';
```

Comment:
If we do use the [quasar pagination component](https://quasar.dev/vue-components/pagination), we will not need this.